client/app/telecom/telephony/billingAccount/billing/tollfreeHistory/telecom-telephony-billing-account-billing-tollfree-history.controller.js

```diff
@@ -84,15 +59,10 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingTo
     ===============================*/
 
     self.download = function (consumption) {
-        consumption.downloading = true;
-
         return self.fetchFile(consumption).then(function (info) {
             $window.location.href = info.url;
-        }).catch(function (error) {
-            Toast.error([$translate.instant("telephony_group_billing_tollfree_history_download_error"), (error.data && error.data.message) || ""].join(" "));
-            return $q.reject(error);
-        }).finally(function () {
-            consumption.downloading = false;
+        }, function (err) {
+            return new ToastError([$translate.instant("telephony_group_billing_tollfree_history_download_error"), (err.data && err.data.message) || ""].join(" "));
```

Comment:
This is not how `ToastError` works. It takes as arguments an error object AND a translation key, not a string